### ISSUE TITLE
Accordion - prop for heading level

### ISSUE THREAD

**Issue by jgiardino** [patternfly/patternfly-react#1881]
A new prop `headingLevel` should be added to the Accordion component to allow the user to specify heading levels. Supported values should be `h1` – `h6`.

Note: prop name and values are consistent with Title component.

**Comment by jgiardino** [comment]
Some extra notes on this:

1. As an example, if I set `headingLevel="h2"`, I should see the following results in the html:
    ```
    <dl class="pf-c-accordion">
      <dt>
        <h2>
          <button id="ex-toggle1" class="pf-c-accordion__toggle" variant="plain" aria-expanded="false">
            ...
          </button>
        </h2>
      </dt>
      <dd>...</dd>
    </dl>
    ```
    Whatever value I specify for `headingLevel` should be used for the heading element that wraps the button. 

2. My best guess is that this should get applied to the parent component, and applied to all the headings in that component. For example:
    `<Accordion headingLevel="h2">`
  My reason for this is that these headings are all peers and therefore should have the same heading level. And it simplifies setting this value to only have to define it once. If the use case exists to have different heading levels, then really the right structure for this type of heading outline is nested accordions.

3. The only other component where we have a headingLevel prop (that I have found) is the Title component. This component uses `"h1"` as a default if `headingLevel` is not defined. I'd say let's go with that for consistency.
